### PR TITLE
Feature/snapshot

### PR DESCRIPTION
**Pull request by 9renpoto**
### TL;DR

add snapshot spec

### checklist

- [x] CHECK SUMMERY.md
- [x] CHECK GLOSSARY.md

### CODE REVIEW

**Comment by tsuyukimakoto** [comment]
[LGTM](http://lgtm.in/i/GV2L6wo0q)

[:+1:](http://lgtm.in/u/GV2L6wo0q) [:-1:](http://lgtm.in/r/GV2L6wo0q)

File: en/web/guide/snapshot.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Form (When session information is remaining)
+
+# Requirements
+
+Followings are required for virtual page.
+
+- [Create virtual page](../devguide/virtualpage.html)
+- [Specify virtual page and start to tracking](../devguide/javascript/api/create.html)
+
+## Merit/Demerit
```
Comment (Surgo) [inline en/web/guide/snapshot.md]: pros/cons がいい

**Comment by Surgo** [comment]
[LGTM](http://lgtm.in/i/yVQfaOteq)

[:+1:](http://lgtm.in/u/yVQfaOteq) [:-1:](http://lgtm.in/r/yVQfaOteq)

File: en/web/guide/snapshot.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Requirements
+
+Followings are required for virtual page.
+
+- [Create virtual page](../devguide/virtualpage.html)
+- [Specify virtual page and start to tracking](../devguide/javascript/api/create.html)
+
+## pros/cons
+
+Merit and Demerit of the virtual page.
```
Comment (Surgo) [inline en/web/guide/snapshot.md]: :oh:

**Pull request merged**